tools/ml-dataprep/dataprep.py

```diff
@@ -0,0 +1,249 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module containing the main class that generates the training and validation
+ML datasets from a BigQuery table."""
+
+
+import datetime
+import queries
+import bigquery
+
+
+TIMESTAMP_FORMAT = '%Y%m%d%H%M%S'
+TRAINING_DATASET = 'training'
+VALIDATION_DATASET = 'validation'
+TEMP_TABLE_SUFFIX = 'temp'
+COLUMNS_SEPARATOR = ','
+
+
+class DataPreparator:
+    # pylint: disable=too-many-instance-attributes
+
+    """Data preparation class which generates training and validation ML
```

Comment:
Docstring for the class init variables are required also

Reply:
Done in 510475852eade28e9aa92c24a39bacc7624f8a33